migration/v0.43/apply/10-kube-prometheus-stack.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    new_version=66.1.1
+    clusters=("${CK8S_CLUSTER}")
+    if [[ "${CK8S_CLUSTER}" == "both" ]]; then
+      clusters=("wc" "sc")
+    fi
+
+    for cluster in "${clusters[@]}"; do
+      current_version=$(helm_do "${cluster}" get metadata -n monitoring kube-prometheus-stack -ojson | jq '.version' | tr -d '"')
+
+      log_info "  - Checking if kube-promethes-stack needs to be upgraded"
+      if [[ ! "${current_version}" < "$(echo -e "${new_version}\n${current_version}" | sort -V | tail -n1)" ]]; then
```

Comment:
Question: Does this comparison work with versions?

Wouldn't this just need to be

```suggestion
      if [[ "${current_version}" != "${new_version}" ]]; then
```

Reply:
The comparison should work in most cases since it sorts lexicographically and the ksp helm release will be a couple of major versions behind in most cases. But I like your suggestion, then it should be possible to do downgrade as well. PTAL https://github.com/elastisys/compliantkubernetes-apps/pull/2381/commits/78f5a5c3790976d2308b8a42731fc1bc11e95c6d